doc/usecases/cli/UC_get_config.md

```diff
@@ -0,0 +1,22 @@
+# Use Case: cli get configuration
+
+## Summary
+
+- **Scope:** CLI use
+- **Level:** Developer Goal, UI
+- **Actors:** CLI user
+- **Brief:** User reads data for a given keyname
+
+## Scenarios
+
+- **Precondition:** -
+- **Main success scenario:**
+  - Backend is asked for the data for the given keyname
```

Comment:
```suggestion
  - The CLI user queries data from the backend by providing a keyname.
```
The scenario should mention the actors (CLI user in this case) and their actions.